lib/functions.zsh

```diff
@@ -63,6 +63,32 @@ function try_alias_value() {
 }
 
 #
+# Alias only if not already a command/function/alias
+# Use option ALLOW_OVERRIDE_ALIASES to force overriding
+#
+# Arguments:
+#    1. my_alias='my alias value'
+#       Same argument you would pass to the "alias" built-in
+# Return value:
+#    0 if the alias was safely created, 73 otherwise
+#
+function safe_alias() {
+    local alias lhs rhs
```

Comment:
Would this not make the alias local to the function? Sorry but I do not know this well.